src/lib/rest/rest.cpp

```diff
@@ -871,6 +871,11 @@ static std::string apiVersionGet(const char* path)
     return "v2";
   }
 
+  if (strcmp(path, "/admin/log") == 0)
+  {
+    return "v2";
```

Comment:
Actually, /admin is orthogonal to v1 or v2... Maybe "admin" should be the right one in this case, but not sure (actually, not sure in which places this apiVersionGet is being used in the case of /admin/log).